### DISCUSSION TITLE
inertia-link

### DISCUSSION THREAD

**Discussion by DavidGarrido** [inertiajs/inertia#981] [Ideas]
Hola, hice una instalacion limpia de inertia y no me funciona la etiqueta inertia-link me doy cuenta que en welcome.vue ahora hace referencia Link a en que actualizacion se hicieron esos cambios?

**Comment by RobertBoes** [comment]
Global component registration was removed in [inertia-vue@0.7.0](https://inertiajs.com/releases/inertia-vue-0.7.0-2021-07-13) and [inertia-vue3@0.5.0](https://inertiajs.com/releases/inertia-vue3-0.5.0-2021-07-13), hope that answers your question

Ps. next time try to post your questions in English, had to use translate to know what you were asking 🙂
